apps/dashboard/src/components/command-palette/commands/navigation-commands.tsx

```diff
@@ -108,17 +107,9 @@ export function useNavigationCommands(context: CommandExecutionContext): Command
     );
   }
 
-  if (isTranslationEnabled) {
-    commands.push(
-      createNavigationCommand(
-        'nav-translations',
-        'Translations',
-        ROUTES.TRANSLATIONS,
-        <RiTranslate2 />,
-        () => isTranslationEnabled
-      )
-    );
-  }
+  commands.push(
+    createNavigationCommand('nav-translations', 'Translations', ROUTES.TRANSLATIONS, <RiTranslate2 />)
+  );
```

Comment:
@ChmaraX we should still have this feature flag or switch it to the ENTERPRISE feature flag for community edition on the front end